packages/opentelemetry-plugin-xml-http-request/src/xhr.ts

```diff
@@ -14,51 +14,22 @@
  * limitations under the License.
  */
 
-import {
-  BasePlugin,
-  hrTime,
-  isUrlIgnored,
-  isWrapped,
-  otperformance,
-  urlMatches,
-} from '@opentelemetry/core';
+import { BasePlugin, hrTime, isWrapped, otperformance, isIgnored } from '@opentelemetry/core';
 import * as types from '@opentelemetry/types';
-import {
-  addSpanNetworkEvent,
-  getResource,
-  parseUrl,
-  PerformanceTimingNames as PTN,
-} from '@opentelemetry/web';
+import { PluginOptions, XMLHttpRequestPluginOptions, IgnoreMatcher } from '@opentelemetry/types';
```

Comment:
why do you import these if you already have `types` above?

Reply:
Good catch. I think my editor did auto import